bundles/SimpleBackendSearchBundle/src/DependencyInjection/PimcoreSimpleBackendSearchExtension.php

```diff
@@ -43,6 +43,7 @@ public function prepend(ContainerBuilder $container): void
             );
 
             $loader->load('admin-classic.yaml');
+            $loader->load('doctrine_migrations.yaml');
```

Comment:
@bramstroker Why should this only be loaded if 'pimcore_admin' extension is available? I think this should be moved to the `load()` method

Reply:
You are completely right. Nice catch. It's fixed now.